beacon-chain/sync/validate_aggregate_proof.go

```diff
@@ -265,7 +266,10 @@ func (s *Service) hasSeenAggregatorIndexEpoch(epoch primitives.Epoch, aggregator
 func (s *Service) setAggregatorIndexEpochSeen(epoch primitives.Epoch, aggregatorIndex primitives.ValidatorIndex) {
 	s.seenAggregatedAttestationLock.Lock()
 	defer s.seenAggregatedAttestationLock.Unlock()
-	b := append(bytesutil.Bytes32(uint64(epoch)), bytesutil.Bytes32(uint64(aggregatorIndex))...)
+
+	b := make([]byte, 64)
+	binary.BigEndian.PutUint64(b[24:], uint64(epoch))
+	binary.BigEndian.PutUint64(b[56:], uint64(aggregatorIndex))
```

Comment:
These slices offsets look wrong, and it seems like this is changing the encoding from `LittleEndian` to `BigEndian`? I think it should be:
```
	b := make([]byte, 64)
	binary.LittleEndian.PutUint64(b, uint64(epoch))
	binary.LittleEndian.PutUint64(b[32:], uint64(aggregatorIndex))
```